packages/cfd/src/Containers/__tests__/cfd-change-password-confirmation.spec.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { render, screen, fireEvent } from '@testing-library/react';
+import ChangePasswordConfirmation from '../cfd-change-password-confirmation';
+
+it('should render cfd-change-password-confirmation component', () => {
+    render(<ChangePasswordConfirmation />);
+    expect(screen.getByTestId('cfd_change_password_form')).toBeInTheDocument();
+});
+
+it('should show icon and buttons', () => {
+    render(<ChangePasswordConfirmation />);
+
+    const icon = screen.getByTestId('cfd_change_password_confirmation_icon');
+    expect(icon).toBeInTheDocument();
```

Comment:
instead of adding testid and testing, maybe we can mock Icon component

Reply:
Alrightss On it